telegram.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Get telegram user data
+ * 
+ * Gets telegram user data from cookie and save it to array
+ * 
+ * @return void
+ *
+ * @author Telegram
+ *
+ *
+ * @since 0.1
+ */
+function getTelegramUserData() {
+	if (isset($_COOKIE['tg_user'])) {
+		$auth_data_json = urldecode($_COOKIE['tg_user']);
+		 $auth_data = json_decode($auth_data_json, true);
+		return $auth_data;
+	}
+	return false;
+}
+/** 
+ * Check if data is from telegram
+ * 
+ * This checks if the data provides is from telegram. It includes a Fix for firefox
+ * 
+ * @param mixed $auth_data The Authentication Data
+ * 
+ * @return $auth_data
+ * 
+*/
+function checkTelegramAuthorization($auth_data) {
+	$check_hash = $auth_data['hash'];
+	unset($auth_data['hash']);
+	$data_check_arr = [];
+	foreach ($auth_data as $key => $value) {
+	 // $data_check_arr[] = $key . '=' . $value;
```

Comment:
Nit: remove :slightly_smiling_face:

Reply:
You mean line 36? This is how it should be done, but in firefox it doesn't work. But i will remove it tomorrow.

Reply:
It's just that the commented out code doesn't look great - optionally you could just write that it is the proper way to do it but workaround is used - that would be a lot better

Reply:
Yeah, will look into it tomorrow!